dask/dataframe/core.py

```diff
@@ -3122,13 +3122,24 @@ def nunique(self, split_every=None):
 
     @derived_from(pd.Series)
     def value_counts(
-        self, sort=None, ascending=False, dropna=None, split_every=None, split_out=1
+        self,
+        sort=None,
+        ascending=False,
+        dropna=None,
+        normalize=False,
+        split_every=None,
+        split_out=1,
     ):
         """
         Note: dropna is only supported in pandas >= 1.1.0, in which case it defaults to
         True.
         """
-        kwargs = {"sort": sort, "ascending": ascending}
+        aggregate_kwargs = {
+            "sort": sort,
+            "ascending": ascending,
+            "normalize": normalize,
+        }
```

Comment:
Does this approach work if `split_out`>1 ?  It seems like each output partition will be normalized independently.